dev/camdenhells/REQUIREMENTS.md

```diff
@@ -55,34 +55,37 @@ For reviews and updates:
 ## 3.1 Android and OpenGL ES integration
 
 1. Provide (major and minor) version query for the current GL ES render context
-2. Allow multiple `GlSurfaceViews` per Activity ? [2.1]
-  * Is this actually worthwhile?
-  * Only one render thread/context/Renderer can exist at once, so SurfaceViews are mutually exclusive 
-    * E.g. `ViewPager` support (per #1619)
-    * Is timing of render context switch still based on window attachment?
-3. Wallpaper surface view multisampling fix (#1559)
-4. Provide query/configuration constants (bit flags) for all nVidia GL ES extensions (#1742) [2.1]
-  * Do any other vendors or extension categories need this?
+2. Decouple Android views from the EGL context to allow offline rendering/initialization
+3. ~Allow multiple `GlSurfaceViews` per Activity ? [2.1]~
```

Comment:
If we can't or won't support multiple GlSurfaceViews per Activity (presumably with only one unpaused/running at a time), then should #1619 be declined? How else can there be multiple renderers?

Reply:
After some thought I think I misunderstood this. I will update.